src/e2e/java/teammates/e2e/cases/InstructorHomePageE2ETest.java

```diff
@@ -114,7 +114,7 @@ public void testAll() {
         copiedSession.setTimeZone(otherCourse.getTimeZone());
         homePage.copySession(courseIndex, sessionIndex, otherCourse, newName);
 
-        homePage.waitForConfirmationModalAndClickOk();
+        // homePage.waitForConfirmationModalAndClickOk();
```

Comment:
Can I check why this line is commented out?

Reply:
updated!